src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ButtonCommand.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Windows.Input;
+
+namespace NuGet.Options
+{
+    internal class ButtonCommand : ICommand
```

Comment:
I may be wrong but, I believe you can remove this class and achieve the same result using CommandBinding directly in XAML control.

Buttons in both dialogs (option dialog and add source mapping dialog) can map to method actions using XAML CommandBinding construc.

An example can be found in PackageManagerControl for search command:

https://github.com/NuGet/NuGet.Client/blob/3ec6f02943fa8e70ea224eb31dd94288dbf179d0/src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml#L34-L36

Ping me if you have more questions.